scripts/src/seed-database.ts

```diff
@@ -0,0 +1,367 @@
+import { Timestamp } from "firebase-admin/firestore";
+import admin, { initializeApp } from "./admin";
+
+initializeApp();
+
+const seedDatabase = async () => {
+  const db = admin.firestore();
+
+  await db.runTransaction(async (transaction) => {
+    // Create templates
+    transaction.set(db.collection("templates").doc(), DEFAULT_TEMPLATE);
+
+    // Create participants
+    DEFAULT_PARTICIPANTS.forEach((participant, index) => {
+      transaction.set(
+        db.collection("participants").doc(`participant-${index}`),
+        participant
+      );
+    });
+
+    // Create experiment
+    transaction.set(
+      db.collection("experiments").doc("default-experiment-id"),
+      DEFAULT_EXPERIMENT
+    );
+
+    // Create synchronization collections
+    // Create their progression data in a separate collection (for synchronization purposes)
+    transaction.set(
+      db.collection("participants_progressions").doc("default-experiment-id"),
+      {
+        experimentId: "default-experiment-id",
+        progressions: {
+          "participant-0": "1. Agree to the experiment and set your profile",
+          "participant-1": "1. Agree to the experiment and set your profile",
+          "participant-2": "1. Agree to the experiment and set your profile",
+        },
+      }
+    );
+
+    // Create the `ready_to_end_chat` entry for synchronization purposes
+    transaction.set(
+      db.collection("participants_ready_to_end_chat").doc("default-chat-id"),
+      {
+        chatId: "default-chat-id",
+        readyToEndChat: {
+          "participant-0": false,
+          "participant-1": false,
+          "participant-2": false,
+        },
+        currentPair: 0,
+      }
+    );
+  });
+
+  console.log("Done !");
+};
+
+// ********************************************************************************************* //
+//                                         SEEDER DATA                                           //
+// ********************************************************************************************* //
+
+// Note that this data is hardcoded for now, because the db structure will change in order to be more practical.
+
+const DEFAULT_STAGES = [
```

Comment:
It's probably good to have this somewhere general, so it can be used in tests, and here, and also to make sure the constant is type-checked; e.g. 

const DEFAULT_STAGES: <the type of a stage>[] = [ ...